src/components/AddNewBook.js

```diff
@@ -6,16 +6,18 @@ import { addBook } from '../redux/Books/books';
 const AddNewBook = () => {
   const [inputValues, setInputValues] = useState({
     booktitle: '',
+    author: '',
     category: '',
   });
   const dispatch = useDispatch();
 
   const submitBookToStore = (e) => {
     e.preventDefault();
     const id = uuidv4();
-    const { booktitle, category } = inputValues;
+    const { booktitle, author, category } = inputValues;
     const newBook = {
       booktitle,
+      author,
       id,
       category,
     };
```

Comment:
- [OPTIONAL] In the future, let's make sure to add all your changes relevant to the current project in the commits of this PR. It is not currently possible to see them here. As an example this `submitBookToStore `function is incomplete